lighthouse-cli/performance-experiment/server.js

```diff
@@ -94,8 +97,19 @@ function requestHandler(request, response) {
 
 function reportRequestHandler(request, response) {
   try {
+    const id = request.parsedUrl.query.id || defaultReportId;
+
+    const reportsMetadata = Object.keys(database.timeStamps).map(key => {
+      const generatedTime = database.timeStamps[key];
+      return {url: database.url, reportHref: `/?id=${key}`, generatedTime};
```

Comment:
I think it's better to define URL format only in perf-x server. Because if we somehow want to change the URL format in the future, we only need to make changes in perf-x server. Also, it's possible for other clients can pass reports in a different URL format to the generator if we build URL in the server side.